src/system/ThemeModule/EventListener/DefaultPageVarSetterListener.php

```diff
@@ -59,10 +59,10 @@ public function setDefaultPageVars(GetResponseEvent $event)
         // set some defaults
         $this->pageVars->set('lang', $event->getRequest()->getLocale()); //\ZLanguage::getLanguageCode());
         $this->pageVars->set('langdirection', 'ltr'); //\ZLanguage::getDirection());
-        $this->pageVars->set('title', $this->variableApi->getSystemVar('defaultpagetitle'));
+        $this->pageVars->set('title', $this->variableApi->getSystemVar('defaultpagetitle_'.\ZLanguage::getLanguageCode()));
```

Comment:
I am not in favor of adding more usage of `ZLanguage` to Core-2.0 (like this listener). you can see that I have already replaced some of it. You can use `$event->getRequest()->getLocale()` instead